Apps/Medication/src/Delegates/RestMedStatementDelegate.cs

```diff
@@ -189,28 +174,26 @@ public async Task<bool> ValidateProtectiveWordAsync(string phn, string? protecti
         /// <param name="hdid">The HDID of the user querying.</param>
         /// <param name="ipAddress">The IP of the user querying.</param>
         /// <returns>The hash of the protective word response or null if not set.</returns>
-        private async Task<IHash> GetProtectiveWord(string phn, string hdid, string ipAddress)
+        private async Task<HmacHash> GetProtectiveWordHash(string phn, string hdid, string ipAddress)
```

Comment:
This is a fairly significant downgrade of the re-usability of the code.  Should we proceed with the DistributedCache or revert to the old implementation?

Reply:
Reverted to old implementation